plugin/supervisor.go

```diff
@@ -23,10 +23,10 @@ type supervisor struct {
 }
 
 func newSupervisor(pluginInfo *model.BundleInfo, parentLogger *mlog.Logger, apiImpl API) (retSupervisor *supervisor, retErr error) {
-	supervisor := supervisor{}
+	supervisorInstance := supervisor{}
```

Comment:
This the inverse of your earlier argument:
> I don't like activePluginStruct. It is a struct, why do you need to state this in the name?

Given that we /have/ to rename something in this case to avoid shadowing, renaming the `struct` seems better than tacking on a redundant `Instance`. 3/5 on this one, and hence why I suggest `activePlugin<Something>` vs. renaming `activePlugin` for the same reasons.